bot/exts/evergreen/catify.py

```diff
@@ -0,0 +1,44 @@
+import random
+from typing import Optional
+
+from discord.ext import commands
+
+from bot.constants import Cats
+
+
+class Catify(commands.Cog):
+    """Cog for the catify command."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["ᓚᘏᗢify", "ᓚᘏᗢ"])
+    async def catify(self, ctx: commands.Context, *string: Optional[str]) -> None:
+        """Catifies your nickname or a given string."""
+        if string == ():
+            display_name = ctx.author.name
```

Comment:
We should use the display name of the user so that if they have an in-server nickname that is the text that is used.
```suggestion
            display_name = ctx.author.display_name
```